dio/lib/src/transformers/fused_transformer.dart

```diff
@@ -88,9 +89,12 @@ class FusedTransformer extends Transformer {
     if (isJsonContent && decodedResponse != null) {
       // slow path decoder, since there was a custom decoder specified
       return jsonDecode(decodedResponse);
-    } else if (decodedResponse != null) {
+    } else if (customResponseDecoder != null) {
       return decodedResponse;
     } else {
+      if (responseBytes.isEmpty) {
```

Comment:
Any references?

Reply:
Sorry, did not mean to commit that. Is unnecessary, utf8.decode returns an empty string anyway on an empty bytes list.